2022/project_idea.md

```diff
@@ -1,106 +1,116 @@
-## Project’s scope
-The Arviz documentation project will
-* Evaluate multiple alternatives to generate this Bayesian workflow resource.
+### Project’s scope
+The Arviz documentation project scope includes the following:
+* Evaluate multiple alternatives to generate this Bayesian workflow resource. 
   The ideal requirements are:
-  - The bulk of the explanation/prose content will be common.
-    The _multiple_ pages for each programming language and PPL need to be combined
-    with the _single_ explanation source of truth and rendered.
-  - The code examples must be written in executable files and store its outputs
-    for rendering automatically (and without extra code).
-    This will not be a static resource and will need to be reexecuted regularly.
-  - The explanation content must be written in a format that supports internationalization.
-    Ideally, code annotations or comments would too, but they can be avoided
-    or kept in English.
-  - Changes to the explanation content should be reflected on the website automatically
-    when merged. Reexecuting and updating code outputs can have manual steps.
-* Set up all a proof of concept in the [bayesian-workflow](https://github.com/arviz-devs/bayesian-workflow) repo
-  together with all CI/CD necessary for updating the content.
-* Regular presentations to the ArviZ team about the writing workflow
+- The bulk of the explanation/prose content will be common. 
+  The _multiple_ pages for each programming language and PPL need to be combined 
+  with the single explanation source of truth and rendered.
+- The code examples must be written in executable files and store their outputs
+  for rendering automatically (and without extra code). 
+  This will not be a static resource and will need to be updated regularly.
+- The explanation content must be written in a format that supports internationalization. 
+  Ideally, code annotations or comments would do but they can be avoided or kept in English.
+
+- Changes to the explanation content should be reflected on the website automatically
+  when merged. Reexecuting and updating code outputs can have manual steps.
+* Set up  a proof of concept in the [bayesian-workflow](https://github.com/arviz-devs/bayesian-workflow) repo
+* together with all CI/CD necessary for updating the content.
+* Regular presentations to the ArviZ team about the writing workflow.
 * Write documentation about maintaining the repository.
 * Define the style and structure of the resource.
-* Start adding some example/template pages (together with the ArviZ educational lead).
-  For example, a quick way to add content and motivate other people to
-  collaborate would be adapting some case studies from the documentations
-  of PPLs.
+* Start adding some example/template pages (together with the ArviZ educational lead). 
+* For example, a quick way to add content and motivate other people to 
+* collaborate would be to adopt some case studies from the documentation 
+* of PPLs.
```

Comment:
```suggestion
* Start adding some example/template pages (together with the ArviZ educational lead). 
  For example, a quick way to add content and motivate other people to 
  collaborate would be to adopt some case studies from the documentation 
  of PPLs.
```

Keep this as text of the bullet point defined in the first line, not each line a different bullet point.